Wiki Reorg: Cards Learn Section (#4964)
w3f/polkadot-wiki: docs/learn/learn-parachains-protocol.md

id: learn-parachains-protocol
title: Parachains' Protocol Overview
sidebar_label: Protocol Overview

> description: Actors and Protocols shaping Polkadot's Block Road to Finality.

Reviewer: ```suggestion
description: Actors and Protocols involved in Polkadot and its Parachains' Block Finality.
```
